### ISSUE TITLE
Junction reads or also Junction fragments?

### ISSUE THREAD

**Issue by glopatriello** [Xinglab/rmats-turbo#236]
Hi there!

I am performing an analysis with rMATS on our RNA-seq samples. I was wondering if the algorithm takes into account also the junction fragments (mates of the same fragment that maps in two different exons), in addition to junction reads(spliced reads mapping across a junction).

Best regards,
Giulia

**Comment by EricKutschera** [comment]
No, rMATS will not check if two mates map to different exons. rMATS will check for the proper pair flag if `-t paired`. rMATS will check that the aligned strands of the mates are as expected if `--libType fr-firststrand` or `--libType fr-secondstrand`. Besides those checks, rMATS treats each mate separately
